### ISSUE TITLE
Improve connector documentation

### ISSUE THREAD

**Issue by shaunagm** [move-coop/parsons#269]
The connector documentation to be more consistent.  Each connector needs, in order:

1) an overview, which explains the service that the connector connects to, and summarizes what subsets of endpoints currently are covered.  
2) a detailed guide to getting authentication info.  
3) a "quick start guide" that demonstrates instantiating the connector and using a method.  

See [the action kit connector documentation](https://move-coop.github.io/parsons/html/action_kit.html) for an example of what we're looking for.

This doesn't need to be done by a single person in a single PR - feel free to grab a connector by leaving a comment below and submit the updated docs as its own PR.

Here's the current status of the connector docs (thanks to @rgriff23 for the audit!):

| Connector | Overview | Authentication Guide | Quickstart | Notes | 
| -------------- | :--------- | ----------: | ----------: | ----------: | 
| Action Kit | ✔️ | ✔️ | ✔️ | |
| Action Network |  ✔️| ✔️ | ✔️ | |
| Airtable | ✔️ | ✔️ |  ✔️ | |
| AWS: ALL |   ||| needs a global overview at the top to orient the reader |
| AWS: Lambda | to add | to add | to add | |
| AWS: S3 | ✔️ | ✔️| ✔️ | |
| AWS: Redshift | ✔️ |  ✔️| ✔️| |
| Azure: Blob Storage | ✔️ |  ✔️| ✔️| |
| Bill.com | ✔️ |  ✔️| ✔️| |
| Braintree |  ✔️ | ✔️ | ✔️ | |
| Civis |  ✔️ | ✔️ | ✔️ | |
| Copper |  ✔️ | ✔️ | ✔️ | |
| CrowdTangle |  ✔️ | ✔️ | ✔️ | |
| Databases: ALL | ✔️ | | |  |
| Databases: MySQL | ✔️ | ✔️ | ✔️ | |
| Databases: Postgres | ✔️ | ✔️ | ✔️ | |
| Databases: Database Sync | ✔️ | ✔️  | ✔️ | |
| Facebook Ads |  ✔️ | ✔️ | ✔️ | |
| FreshDesk |  ✔️ | ✔️ | ✔️ | |
| Github | ✔️ | ✔️  | ✔️ | |
| Google: ALL | | | | needs a global overview at the top to orient the reader |
| Google: Big Query | ✔️| to add | to add | |
| Google: CloudStorage | needs improvement| to add| to add | |
| Google: Civic | ✔️ | to add| to add| |
| Google: Google Sheets | needs improvement|needs improvement | to add | |
| Hustle | ✔️ |  ✔️ | ✔️ | |
| MailChimp | ✔️ | ✔️  | ✔️ | |
| Mobilize America |  ✔️ | ✔️ | ✔️ | |
| New/Mode | ✔️ | ✔️ | ✔️ | |
| Ngpvan |✔️ |✔️ |✔️ | |
| PDI |✔️ |✔️ |✔️ | |
| Phone2Action | ✔️ |✔️ |✔️ | |
| Redash | ✔️ |✔️ |✔️ | |
| Rock the Vote | ✔️ |✔️ |✔️ | |
| Salesforce | ✔️ |✔️ |✔️ | |
| SFTP | ✔️ |✔️ |✔️ | |
| TargetSmart | ✔️ |✔️ |✔️ | |
| Twilio |  ✔️ | ✔️ | ✔️ | |
| TurboVote |  ✔️ | ✔️ | ✔️ | |
| Zoom| ✔️ | ✔️ | ✔️ | |

**Comment by shaunagm** [comment]
Some potential documentation updates:

- connector documentation should be more consistent, with each connector including a "how to get authentication info" section and a "quick start guide", and the quick start guide should make sure to explain the passing in of authentication info, which some of the examples currently don't
- the contributing.md file should maybe be in the top level, to be easier to find?
- the contributing.md file should include more explicit explanations of concepts that might be new to people, like understanding what they see when they run pytest, or setting up a virtualenv if they don't have the right python version (or even if they do)
- our documentation, esp install instructions, should be tested on Windows and Linux as well as Mac

**Comment by shaunagm** [comment]
There should definitely be a "how to write your first connector" guide.  I'm working on making my first connector now, and here are some of the things I've run into so far that could be addressed in such a guide:

- what the file-level structure should be - I'm guessing we want a folder, an init.py, and a file with the same name as the folder, since that's what currently exists.  this guide would be a good place to briefly explain how python uses init.py to manage imports, in case folks are interested (and so they don't forget the init.py, because now they understand why it's important)
- the within-file structure - logging, class and method naming conventions, etc.  How should classes be initialized?  When you want to do X, what structure should you follow?  We can link to existing classes/methods, like "If you want to get a csv file from a server and turn it into a Parsons table, follow this pattern:"
- not being sure what needs to be done beyond the connector itself (so far I've got: update requirements.txt with added dependencies; update docs; update tests - but perhaps I'm missing steps?)

**Comment by shaunagm** [comment]
Training:

These can be imagined both as online, live trainings, but also as videos or written guides, once we've gotten the kinks worked out.

- get started using parsons (explains basic concepts, shows common workflows); different versions for the "new to coding/python" and the "comfortable with coding/python" users
- advanced parson usage (common but significantly more complex workflows)
- how to contribute to parsons; again, different versions for the "new to coding/python" and the "comfortable with coding/python" contributors
- "understanding the progressive data/tech space" (with a special focus on Civis, perhaps?)

Topics to cover, esp for people who are new to Python:

- what are environmental variables, why do we use them, how do you set them
- what are unit tests and how do you write them
- how Parsons does syncing

**Comment by shaunagm** [comment]
Misc stuff to support community growth:

- we should be really explicit about our communication channels so people know how to talk to us, where to ask for help, how to report bugs, feature requests, etc
- we should have a public-facing roadmap
- we should work on being able to articulate what kinds of things belong in parsons and which don't, and possibly incorporating some of the helper/glue code people have written into parsons
- it would be nice to do some kind of regular project night (or afternoon, etc) where the community comes together at the same time to chat, work on things, etc.

**Comment by shaunagm** [comment]
Note from the install party:

- Some questions about how to tell if Parsons installed properly.  We eventually decided that we should just tell people to run tests, but it raises the question of needing accounts to run a lot of Parsons functionality.  Could we create "dummy" accounts that people could use for testing?

**Comment by SorenSpicknall** [comment]
Expanding on the above:

- We should update the quickstart guides to hit example endpoints/public accounts of vendors, where possible, or use functions that won't error out due to missing environment variables or credential info.

**Comment by railmeat** [comment]
All the errors I got from running pytest.

matt@MattPad parsons (master) $ pytest test/test_redshift.py
========================================================== test session starts ===========================================================
platform darwin -- Python 3.7.3, pytest-5.2.0, py-1.8.2, pluggy-0.13.1
rootdir: /Users/matt/proj/parsons, inifile: pytest.ini
plugins: datadir-1.3.0, requests-mock-1.5.2
collected 44 items                                                                                                                       

test/test_redshift.py .............sssssssssssssssssssssssssssssss                                                                 [100%]

============================================================ warnings summary ============================================================
/Library/Python/3.7/site-packages/googleapis_common_protos-1.52.0-py3.7.egg/google/api/field_behavior_pb2.py:40
  /Library/Python/3.7/site-packages/googleapis_common_protos-1.52.0-py3.7.egg/google/api/field_behavior_pb2.py:40: DeprecationWarning: Call to deprecated create function FileDescriptor(). Note: Create unlinked descriptors is going to go away. Please use get/find descriptors from generated code or query the descriptor_pool.
    dependencies=[google_dot_protobuf_dot_descriptor__pb2.DESCRIPTOR],

/Library/Python/3.7/site-packages/googleapis_common_protos-1.52.0-py3.7.egg/google/api/field_behavior_pb2.py:54
  /Library/Python/3.7/site-packages/googleapis_common_protos-1.52.0-py3.7.egg/google/api/field_behavior_pb2.py:54: DeprecationWarning: Call to deprecated create function EnumValueDescriptor(). Note: Create unlinked descriptors is going to go away. Please use get/find descriptors from generated code or query the descriptor_pool.
    type=None,

/Library/Python/3.7/site-packages/googleapis_common_protos-1.52.0-py3.7.egg/google/api/field_behavior_pb2.py:57
  /Library/Python/3.7/site-packages/googleapis_common_protos-1.52.0-py3.7.egg/google/api/field_behavior_pb2.py:57: DeprecationWarning: Call to deprecated create function EnumValueDescriptor(). Note: Create unlinked descriptors is going to go away. Please use get/find descriptors from generated code or query the descriptor_pool.
    name="OPTIONAL", index=1, number=1, serialized_options=None, type=None

/Library/Python/3.7/site-packages/googleapis_common_protos-1.52.0-py3.7.egg/google/api/field_behavior_pb2.py:60
  /Library/Python/3.7/site-packages/googleapis_common_protos-1.52.0-py3.7.egg/google/api/field_behavior_pb2.py:60: DeprecationWarning: Call to deprecated create function EnumValueDescriptor(). Note: Create unlinked descriptors is going to go away. Please use get/find descriptors from generated code or query the descriptor_pool.
    name="REQUIRED", index=2, number=2, serialized_options=None, type=None

/Library/Python/3.7/site-packages/googleapis_common_protos-1.52.0-py3.7.egg/google/api/field_behavior_pb2.py:63
  /Library/Python/3.7/site-packages/googleapis_common_protos-1.52.0-py3.7.egg/google/api/field_behavior_pb2.py:63: DeprecationWarning: Call to deprecated create function EnumValueDescriptor(). Note: Create unlinked descriptors is going to go away. Please use get/find descriptors from generated code or query the descriptor_pool.
    name="OUTPUT_ONLY", index=3, number=3, serialized_options=None, type=None

/Library/Python/3.7/site-packages/googleapis_common_protos-1.52.0-py3.7.egg/google/api/field_behavior_pb2.py:66
  /Library/Python/3.7/site-packages/googleapis_common_protos-1.52.0-py3.7.egg/google/api/field_behavior_pb2.py:66: DeprecationWarning: Call to deprecated create function EnumValueDescriptor(). Note: Create unlinked descriptors is going to go away. Please use get/find descriptors from generated code or query the descriptor_pool.
    name="INPUT_ONLY", index=4, number=4, serialized_options=None, type=None

/Library/Python/3.7/site-packages/googleapis_common_protos-1.52.0-py3.7.egg/google/api/field_behavior_pb2.py:69
  /Library/Python/3.7/site-packages/googleapis_common_protos-1.52.0-py3.7.egg/google/api/field_behavior_pb2.py:69: DeprecationWarning: Call to deprecated create function EnumValueDescriptor(). Note: Create unlinked descriptors is going to go away. Please use get/find descriptors from generated code or query the descriptor_pool.
    name="IMMUTABLE", index=5, number=5, serialized_options=None, type=None

/Library/Python/3.7/site-packages/googleapis_common_protos-1.52.0-py3.7.egg/google/api/field_behavior_pb2.py:75
  /Library/Python/3.7/site-packages/googleapis_common_protos-1.52.0-py3.7.egg/google/api/field_behavior_pb2.py:75: DeprecationWarning: Call to deprecated create function EnumDescriptor(). Note: Create unlinked descriptors is going to go away. Please use get/find descriptors from generated code or query the descriptor_pool.
    serialized_end=204,

/Library/Python/3.7/site-packages/googleapis_common_protos-1.52.0-py3.7.egg/google/api/field_behavior_pb2.py:104
  /Library/Python/3.7/site-packages/googleapis_common_protos-1.52.0-py3.7.egg/google/api/field_behavior_pb2.py:104: DeprecationWarning: Call to deprecated create function FieldDescriptor(). Note: Create unlinked descriptors is going to go away. Please use get/find descriptors from generated code or query the descriptor_pool.
    file=DESCRIPTOR,

/Library/Python/3.7/site-packages/googleapis_common_protos-1.52.0-py3.7.egg/google/api/http_pb2.py:35
  /Library/Python/3.7/site-packages/googleapis_common_protos-1.52.0-py3.7.egg/google/api/http_pb2.py:35: DeprecationWarning: Call to deprecated create function FileDescriptor(). Note: Create unlinked descriptors is going to go away. Please use get/find descriptors from generated code or query the descriptor_pool.
    serialized_pb=b'\n\x15google/api/http.proto\x12\ngoogle.api"T\n\x04Http\x12#\n\x05rules\x18\x01 \x03(\x0b\x32\x14.google.api.HttpRule\x12\'\n\x1f\x66ully_decode_reserved_expansion\x18\x02 \x01(\x08"\x81\x02\n\x08HttpRule\x12\x10\n\x08selector\x18\x01 \x01(\t\x12\r\n\x03get\x18\x02 \x01(\tH\x00\x12\r\n\x03put\x18\x03 \x01(\tH\x00\x12\x0e\n\x04post\x18\x04 \x01(\tH\x00\x12\x10\n\x06\x64\x65lete\x18\x05 \x01(\tH\x00\x12\x0f\n\x05patch\x18\x06 \x01(\tH\x00\x12/\n\x06\x63ustom\x18\x08 \x01(\x0b\x32\x1d.google.api.CustomHttpPatternH\x00\x12\x0c\n\x04\x62ody\x18\x07 \x01(\t\x12\x15\n\rresponse_body\x18\x0c \x01(\t\x12\x31\n\x13\x61\x64\x64itional_bindings\x18\x0b \x03(\x0b\x32\x14.google.api.HttpRuleB\t\n\x07pattern"/\n\x11\x43ustomHttpPattern\x12\x0c\n\x04kind\x18\x01 \x01(\t\x12\x0c\n\x04path\x18\x02 \x01(\tBj\n\x0e\x63om.google.apiB\tHttpProtoP\x01ZAgoogle.golang.org/genproto/googleapis/api/annotations;annotations\xf8\x01\x01\xa2\x02\x04GAPIb\x06proto3',

/Library/Python/3.7/site-packages/googleapis_common_protos-1.52.0-py3.7.egg/google/api/http_pb2.py:62
  /Library/Python/3.7/site-packages/googleapis_common_protos-1.52.0-py3.7.egg/google/api/http_pb2.py:62: DeprecationWarning: Call to deprecated create function FieldDescriptor(). Note: Create unlinked descriptors is going to go away. Please use get/find descriptors from generated code or query the descriptor_pool.
    file=DESCRIPTOR,

/Library/Python/3.7/site-packages/googleapis_common_protos-1.52.0-py3.7.egg/google/api/http_pb2.py:80
  /Library/Python/3.7/site-packages/googleapis_common_protos-1.52.0-py3.7.egg/google/api/http_pb2.py:80: DeprecationWarning: Call to deprecated create function FieldDescriptor(). Note: Create unlinked descriptors is going to go away. Please use get/find descriptors from generated code or query the descriptor_pool.
    file=DESCRIPTOR,

/Library/Python/3.7/site-packages/googleapis_common_protos-1.52.0-py3.7.egg/google/api/http_pb2.py:92
  /Library/Python/3.7/site-packages/googleapis_common_protos-1.52.0-py3.7.egg/google/api/http_pb2.py:92: DeprecationWarning: Call to deprecated create function Descriptor(). Note: Create unlinked descriptors is going to go away. Please use get/find descriptors from generated code or query the descriptor_pool.
    serialized_end=121,

/Library/Python/3.7/site-packages/googleapis_common_protos-1.52.0-py3.7.egg/google/api/http_pb2.py:119
  /Library/Python/3.7/site-packages/googleapis_common_protos-1.52.0-py3.7.egg/google/api/http_pb2.py:119: DeprecationWarning: Call to deprecated create function FieldDescriptor(). Note: Create unlinked descriptors is going to go away. Please use get/find descriptors from generated code or query the descriptor_pool.
    file=DESCRIPTOR,

/Library/Python/3.7/site-packages/googleapis_common_protos-1.52.0-py3.7.egg/google/api/http_pb2.py:137
  /Library/Python/3.7/site-packages/googleapis_common_protos-1.52.0-py3.7.egg/google/api/http_pb2.py:137: DeprecationWarning: Call to deprecated create function FieldDescriptor(). Note: Create unlinked descriptors is going to go away. Please use get/find descriptors from generated code or query the descriptor_pool.
    file=DESCRIPTOR,

/Library/Python/3.7/site-packages/googleapis_common_protos-1.52.0-py3.7.egg/google/api/http_pb2.py:155
  /Library/Python/3.7/site-packages/googleapis_common_protos-1.52.0-py3.7.egg/google/api/http_pb2.py:155: DeprecationWarning: Call to deprecated create function FieldDescriptor(). Note: Create unlinked descriptors is going to go away. Please use get/find descriptors from generated code or query the descriptor_pool.
    file=DESCRIPTOR,

/Library/Python/3.7/site-packages/googleapis_common_protos-1.52.0-py3.7.egg/google/api/http_pb2.py:173
  /Library/Python/3.7/site-packages/googleapis_common_protos-1.52.0-py3.7.egg/google/api/http_pb2.py:173: DeprecationWarning: Call to deprecated create function FieldDescriptor(). Note: Create unlinked descriptors is going to go away. Please use get/find descriptors from generated code or query the descriptor_pool.
    file=DESCRIPTOR,

/Library/Python/3.7/site-packages/googleapis_common_protos-1.52.0-py3.7.egg/google/api/http_pb2.py:191
  /Library/Python/3.7/site-packages/googleapis_common_protos-1.52.0-py3.7.egg/google/api/http_pb2.py:191: DeprecationWarning: Call to deprecated create function FieldDescriptor(). Note: Create unlinked descriptors is going to go away. Please use get/find descriptors from generated code or query the descriptor_pool.
    file=DESCRIPTOR,

/Library/Python/3.7/site-packages/googleapis_common_protos-1.52.0-py3.7.egg/google/api/http_pb2.py:209
  /Library/Python/3.7/site-packages/googleapis_common_protos-1.52.0-py3.7.egg/google/api/http_pb2.py:209: DeprecationWarning: Call to deprecated create function FieldDescriptor(). Note: Create unlinked descriptors is going to go away. Please use get/find descriptors from generated code or query the descriptor_pool.
    file=DESCRIPTOR,

/Library/Python/3.7/site-packages/googleapis_common_protos-1.52.0-py3.7.egg/google/api/http_pb2.py:227
  /Library/Python/3.7/site-packages/googleapis_common_protos-1.52.0-py3.7.egg/google/api/http_pb2.py:227: DeprecationWarning: Call to deprecated create function FieldDescriptor(). Note: Create unlinked descriptors is going to go away. Please use get/find descriptors from generated code or query the descriptor_pool.
    file=DESCRIPTOR,

/Library/Python/3.7/site-packages/googleapis_common_protos-1.52.0-py3.7.egg/google/api/http_pb2.py:245
  /Library/Python/3.7/site-packages/googleapis_common_protos-1.52.0-py3.7.egg/google/api/http_pb2.py:245: DeprecationWarning: Call to deprecated create function FieldDescriptor(). Note: Create unlinked descriptors is going to go away. Please use get/find descriptors from generated code or query the descriptor_pool.
    file=DESCRIPTOR,

/Library/Python/3.7/site-packages/googleapis_common_protos-1.52.0-py3.7.egg/google/api/http_pb2.py:263
  /Library/Python/3.7/site-packages/googleapis_common_protos-1.52.0-py3.7.egg/google/api/http_pb2.py:263: DeprecationWarning: Call to deprecated create function FieldDescriptor(). Note: Create unlinked descriptors is going to go away. Please use get/find descriptors from generated code or query the descriptor_pool.
    file=DESCRIPTOR,

/Library/Python/3.7/site-packages/googleapis_common_protos-1.52.0-py3.7.egg/google/api/http_pb2.py:281
  /Library/Python/3.7/site-packages/googleapis_common_protos-1.52.0-py3.7.egg/google/api/http_pb2.py:281: DeprecationWarning: Call to deprecated create function FieldDescriptor(). Note: Create unlinked descriptors is going to go away. Please use get/find descriptors from generated code or query the descriptor_pool.
    file=DESCRIPTOR,

/Library/Python/3.7/site-packages/googleapis_common_protos-1.52.0-py3.7.egg/google/api/http_pb2.py:297
  /Library/Python/3.7/site-packages/googleapis_common_protos-1.52.0-py3.7.egg/google/api/http_pb2.py:297: DeprecationWarning: Call to deprecated create function OneofDescriptor(). Note: Create unlinked descriptors is going to go away. Please use get/find descriptors from generated code or query the descriptor_pool.
    fields=[],

/Library/Python/3.7/site-packages/googleapis_common_protos-1.52.0-py3.7.egg/google/api/http_pb2.py:301
  /Library/Python/3.7/site-packages/googleapis_common_protos-1.52.0-py3.7.egg/google/api/http_pb2.py:301: DeprecationWarning: Call to deprecated create function Descriptor(). Note: Create unlinked descriptors is going to go away. Please use get/find descriptors from generated code or query the descriptor_pool.
    serialized_end=381,

/Library/Python/3.7/site-packages/googleapis_common_protos-1.52.0-py3.7.egg/google/api/http_pb2.py:328
  /Library/Python/3.7/site-packages/googleapis_common_protos-1.52.0-py3.7.egg/google/api/http_pb2.py:328: DeprecationWarning: Call to deprecated create function FieldDescriptor(). Note: Create unlinked descriptors is going to go away. Please use get/find descriptors from generated code or query the descriptor_pool.
    file=DESCRIPTOR,

/Library/Python/3.7/site-packages/googleapis_common_protos-1.52.0-py3.7.egg/google/api/http_pb2.py:346
  /Library/Python/3.7/site-packages/googleapis_common_protos-1.52.0-py3.7.egg/google/api/http_pb2.py:346: DeprecationWarning: Call to deprecated create function FieldDescriptor(). Note: Create unlinked descriptors is going to go away. Please use get/find descriptors from generated code or query the descriptor_pool.
    file=DESCRIPTOR,

/Library/Python/3.7/site-packages/googleapis_common_protos-1.52.0-py3.7.egg/google/api/http_pb2.py:358
  /Library/Python/3.7/site-packages/googleapis_common_protos-1.52.0-py3.7.egg/google/api/http_pb2.py:358: DeprecationWarning: Call to deprecated create function Descriptor(). Note: Create unlinked descriptors is going to go away. Please use get/find descriptors from generated code or query the descriptor_pool.
    serialized_end=430,

/Library/Python/3.7/site-packages/googleapis_common_protos-1.52.0-py3.7.egg/google/api/annotations_pb2.py:42
  /Library/Python/3.7/site-packages/googleapis_common_protos-1.52.0-py3.7.egg/google/api/annotations_pb2.py:42: DeprecationWarning: Call to deprecated create function FileDescriptor(). Note: Create unlinked descriptors is going to go away. Please use get/find descriptors from generated code or query the descriptor_pool.
    google_dot_protobuf_dot_descriptor__pb2.DESCRIPTOR,

/Library/Python/3.7/site-packages/googleapis_common_protos-1.52.0-py3.7.egg/google/api/annotations_pb2.py:64
  /Library/Python/3.7/site-packages/googleapis_common_protos-1.52.0-py3.7.egg/google/api/annotations_pb2.py:64: DeprecationWarning: Call to deprecated create function FieldDescriptor(). Note: Create unlinked descriptors is going to go away. Please use get/find descriptors from generated code or query the descriptor_pool.
    file=DESCRIPTOR,

/Library/Python/3.7/site-packages/google_cloud_bigquery-1.21.0-py3.7.egg/google/cloud/bigquery_v2/proto/encryption_config_pb2.py:36
  /Library/Python/3.7/site-packages/google_cloud_bigquery-1.21.0-py3.7.egg/google/cloud/bigquery_v2/proto/encryption_config_pb2.py:36: DeprecationWarning: Call to deprecated create function FileDescriptor(). Note: Create unlinked descriptors is going to go away. Please use get/find descriptors from generated code or query the descriptor_pool.
    google_dot_api_dot_annotations__pb2.DESCRIPTOR,

/Library/Python/3.7/site-packages/google_cloud_bigquery-1.21.0-py3.7.egg/google/cloud/bigquery_v2/proto/encryption_config_pb2.py:64
  /Library/Python/3.7/site-packages/google_cloud_bigquery-1.21.0-py3.7.egg/google/cloud/bigquery_v2/proto/encryption_config_pb2.py:64: DeprecationWarning: Call to deprecated create function FieldDescriptor(). Note: Create unlinked descriptors is going to go away. Please use get/find descriptors from generated code or query the descriptor_pool.
    file=DESCRIPTOR,

/Library/Python/3.7/site-packages/google_cloud_bigquery-1.21.0-py3.7.egg/google/cloud/bigquery_v2/proto/encryption_config_pb2.py:76
  /Library/Python/3.7/site-packages/google_cloud_bigquery-1.21.0-py3.7.egg/google/cloud/bigquery_v2/proto/encryption_config_pb2.py:76: DeprecationWarning: Call to deprecated create function Descriptor(). Note: Create unlinked descriptors is going to go away. Please use get/find descriptors from generated code or query the descriptor_pool.
    serialized_end=261,

/Library/Python/3.7/site-packages/googleapis_common_protos-1.52.0-py3.7.egg/google/api/client_pb2.py:39
  /Library/Python/3.7/site-packages/googleapis_common_protos-1.52.0-py3.7.egg/google/api/client_pb2.py:39: DeprecationWarning: Call to deprecated create function FileDescriptor(). Note: Create unlinked descriptors is going to go away. Please use get/find descriptors from generated code or query the descriptor_pool.
    dependencies=[google_dot_protobuf_dot_descriptor__pb2.DESCRIPTOR],

/Library/Python/3.7/site-packages/googleapis_common_protos-1.52.0-py3.7.egg/google/api/client_pb2.py:60
  /Library/Python/3.7/site-packages/googleapis_common_protos-1.52.0-py3.7.egg/google/api/client_pb2.py:60: DeprecationWarning: Call to deprecated create function FieldDescriptor(). Note: Create unlinked descriptors is going to go away. Please use get/find descriptors from generated code or query the descriptor_pool.
    file=DESCRIPTOR,

/Library/Python/3.7/site-packages/googleapis_common_protos-1.52.0-py3.7.egg/google/api/client_pb2.py:79
  /Library/Python/3.7/site-packages/googleapis_common_protos-1.52.0-py3.7.egg/google/api/client_pb2.py:79: DeprecationWarning: Call to deprecated create function FieldDescriptor(). Note: Create unlinked descriptors is going to go away. Please use get/find descriptors from generated code or query the descriptor_pool.
    file=DESCRIPTOR,

/Library/Python/3.7/site-packages/googleapis_common_protos-1.52.0-py3.7.egg/google/api/client_pb2.py:98
  /Library/Python/3.7/site-packages/googleapis_common_protos-1.52.0-py3.7.egg/google/api/client_pb2.py:98: DeprecationWarning: Call to deprecated create function FieldDescriptor(). Note: Create unlinked descriptors is going to go away. Please use get/find descriptors from generated code or query the descriptor_pool.
    file=DESCRIPTOR,

/Library/Python/3.7/site-packages/google_cloud_bigquery-1.21.0-py3.7.egg/google/cloud/bigquery_v2/proto/model_reference_pb2.py:34
  /Library/Python/3.7/site-packages/google_cloud_bigquery-1.21.0-py3.7.egg/google/cloud/bigquery_v2/proto/model_reference_pb2.py:34: DeprecationWarning: Call to deprecated create function FileDescriptor(). Note: Create unlinked descriptors is going to go away. Please use get/find descriptors from generated code or query the descriptor_pool.
    google_dot_api_dot_annotations__pb2.DESCRIPTOR,

/Library/Python/3.7/site-packages/google_cloud_bigquery-1.21.0-py3.7.egg/google/cloud/bigquery_v2/proto/model_reference_pb2.py:62
  /Library/Python/3.7/site-packages/google_cloud_bigquery-1.21.0-py3.7.egg/google/cloud/bigquery_v2/proto/model_reference_pb2.py:62: DeprecationWarning: Call to deprecated create function FieldDescriptor(). Note: Create unlinked descriptors is going to go away. Please use get/find descriptors from generated code or query the descriptor_pool.
    file=DESCRIPTOR,

/Library/Python/3.7/site-packages/google_cloud_bigquery-1.21.0-py3.7.egg/google/cloud/bigquery_v2/proto/model_reference_pb2.py:80
  /Library/Python/3.7/site-packages/google_cloud_bigquery-1.21.0-py3.7.egg/google/cloud/bigquery_v2/proto/model_reference_pb2.py:80: DeprecationWarning: Call to deprecated create function FieldDescriptor(). Note: Create unlinked descriptors is going to go away. Please use get/find descriptors from generated code or query the descriptor_pool.
    file=DESCRIPTOR,

/Library/Python/3.7/site-packages/google_cloud_bigquery-1.21.0-py3.7.egg/google/cloud/bigquery_v2/proto/model_reference_pb2.py:98
  /Library/Python/3.7/site-packages/google_cloud_bigquery-1.21.0-py3.7.egg/google/cloud/bigquery_v2/proto/model_reference_pb2.py:98: DeprecationWarning: Call to deprecated create function FieldDescriptor(). Note: Create unlinked descriptors is going to go away. Please use get/find descriptors from generated code or query the descriptor_pool.
    file=DESCRIPTOR,

/Library/Python/3.7/site-packages/google_cloud_bigquery-1.21.0-py3.7.egg/google/cloud/bigquery_v2/proto/model_reference_pb2.py:110
  /Library/Python/3.7/site-packages/google_cloud_bigquery-1.21.0-py3.7.egg/google/cloud/bigquery_v2/proto/model_reference_pb2.py:110: DeprecationWarning: Call to deprecated create function Descriptor(). Note: Create unlinked descriptors is going to go away. Please use get/find descriptors from generated code or query the descriptor_pool.
    serialized_end=234,

/Library/Python/3.7/site-packages/google_cloud_bigquery-1.21.0-py3.7.egg/google/cloud/bigquery_v2/proto/standard_sql_pb2.py:34
  /Library/Python/3.7/site-packages/google_cloud_bigquery-1.21.0-py3.7.egg/google/cloud/bigquery_v2/proto/standard_sql_pb2.py:34: DeprecationWarning: Call to deprecated create function FileDescriptor(). Note: Create unlinked descriptors is going to go away. Please use get/find descriptors from generated code or query the descriptor_pool.
    google_dot_api_dot_annotations__pb2.DESCRIPTOR,

/Library/Python/3.7/site-packages/google_cloud_bigquery-1.21.0-py3.7.egg/google/cloud/bigquery_v2/proto/standard_sql_pb2.py:50
  /Library/Python/3.7/site-packages/google_cloud_bigquery-1.21.0-py3.7.egg/google/cloud/bigquery_v2/proto/standard_sql_pb2.py:50: DeprecationWarning: Call to deprecated create function EnumValueDescriptor(). Note: Create unlinked descriptors is going to go away. Please use get/find descriptors from generated code or query the descriptor_pool.
    type=None,

/Library/Python/3.7/site-packages/google_cloud_bigquery-1.21.0-py3.7.egg/google/cloud/bigquery_v2/proto/standard_sql_pb2.py:53
  /Library/Python/3.7/site-packages/google_cloud_bigquery-1.21.0-py3.7.egg/google/cloud/bigquery_v2/proto/standard_sql_pb2.py:53: DeprecationWarning: Call to deprecated create function EnumValueDescriptor(). Note: Create unlinked descriptors is going to go away. Please use get/find descriptors from generated code or query the descriptor_pool.
    name="INT64", index=1, number=2, serialized_options=None, type=None

/Library/Python/3.7/site-packages/google_cloud_bigquery-1.21.0-py3.7.egg/google/cloud/bigquery_v2/proto/standard_sql_pb2.py:56
  /Library/Python/3.7/site-packages/google_cloud_bigquery-1.21.0-py3.7.egg/google/cloud/bigquery_v2/proto/standard_sql_pb2.py:56: DeprecationWarning: Call to deprecated create function EnumValueDescriptor(). Note: Create unlinked descriptors is going to go away. Please use get/find descriptors from generated code or query the descriptor_pool.
    name="BOOL", index=2, number=5, serialized_options=None, type=None

/Library/Python/3.7/site-packages/google_cloud_bigquery-1.21.0-py3.7.egg/google/cloud/bigquery_v2/proto/standard_sql_pb2.py:59
  /Library/Python/3.7/site-packages/google_cloud_bigquery-1.21.0-py3.7.egg/google/cloud/bigquery_v2/proto/standard_sql_pb2.py:59: DeprecationWarning: Call to deprecated create function EnumValueDescriptor(). Note: Create unlinked descriptors is going to go away. Please use get/find descriptors from generated code or query the descriptor_pool.
    name="FLOAT64", index=3, number=7, serialized_options=None, type=None

/Library/Python/3.7/site-packages/google_cloud_bigquery-1.21.0-py3.7.egg/google/cloud/bigquery_v2/proto/standard_sql_pb2.py:62
  /Library/Python/3.7/site-packages/google_cloud_bigquery-1.21.0-py3.7.egg/google/cloud/bigquery_v2/proto/standard_sql_pb2.py:62: DeprecationWarning: Call to deprecated create function EnumValueDescriptor(). Note: Create unlinked descriptors is going to go away. Please use get/find descriptors from generated code or query the descriptor_pool.
    name="STRING", index=4, number=8, serialized_options=None, type=None

/Library/Python/3.7/site-packages/google_cloud_bigquery-1.21.0-py3.7.egg/google/cloud/bigquery_v2/proto/standard_sql_pb2.py:65
  /Library/Python/3.7/site-packages/google_cloud_bigquery-1.21.0-py3.7.egg/google/cloud/bigquery_v2/proto/standard_sql_pb2.py:65: DeprecationWarning: Call to deprecated create function EnumValueDescriptor(). Note: Create unlinked descriptors is going to go away. Please use get/find descriptors from generated code or query the descriptor_pool.
    name="BYTES", index=5, number=9, serialized_options=None, type=None

/Library/Python/3.7/site-packages/google_cloud_bigquery-1.21.0-py3.7.egg/google/cloud/bigquery_v2/proto/standard_sql_pb2.py:68
  /Library/Python/3.7/site-packages/google_cloud_bigquery-1.21.0-py3.7.egg/google/cloud/bigquery_v2/proto/standard_sql_pb2.py:68: DeprecationWarning: Call to deprecated create function EnumValueDescriptor(). Note: Create unlinked descriptors is going to go away. Please use get/find descriptors from generated code or query the descriptor_pool.
    name="TIMESTAMP", index=6, number=19, serialized_options=None, type=None

/Library/Python/3.7/site-packages/google_cloud_bigquery-1.21.0-py3.7.egg/google/cloud/bigquery_v2/proto/standard_sql_pb2.py:71
  /Library/Python/3.7/site-packages/google_cloud_bigquery-1.21.0-py3.7.egg/google/cloud/bigquery_v2/proto/standard_sql_pb2.py:71: DeprecationWarning: Call to deprecated create function EnumValueDescriptor(). Note: Create unlinked descriptors is going to go away. Please use get/find descriptors from generated code or query the descriptor_pool.
    name="DATE", index=7, number=10, serialized_options=None, type=None

/Library/Python/3.7/site-packages/google_cloud_bigquery-1.21.0-py3.7.egg/google/cloud/bigquery_v2/proto/standard_sql_pb2.py:74
  /Library/Python/3.7/site-packages/google_cloud_bigquery-1.21.0-py3.7.egg/google/cloud/bigquery_v2/proto/standard_sql_pb2.py:74: DeprecationWarning: Call to deprecated create function EnumValueDescriptor(). Note: Create unlinked descriptors is going to go away. Please use get/find descriptors from generated code or query the descriptor_pool.
    name="TIME", index=8, number=20, serialized_options=None, type=None

/Library/Python/3.7/site-packages/google_cloud_bigquery-1.21.0-py3.7.egg/google/cloud/bigquery_v2/proto/standard_sql_pb2.py:77
  /Library/Python/3.7/site-packages/google_cloud_bigquery-1.21.0-py3.7.egg/google/cloud/bigquery_v2/proto/standard_sql_pb2.py:77: DeprecationWarning: Call to deprecated create function EnumValueDescriptor(). Note: Create unlinked descriptors is going to go away. Please use get/find descriptors from generated code or query the descriptor_pool.
    name="DATETIME", index=9, number=21, serialized_options=None, type=None

/Library/Python/3.7/site-packages/google_cloud_bigquery-1.21.0-py3.7.egg/google/cloud/bigquery_v2/proto/standard_sql_pb2.py:80
  /Library/Python/3.7/site-packages/google_cloud_bigquery-1.21.0-py3.7.egg/google/cloud/bigquery_v2/proto/standard_sql_pb2.py:80: DeprecationWarning: Call to deprecated create function EnumValueDescriptor(). Note: Create unlinked descriptors is going to go away. Please use get/find descriptors from generated code or query the descriptor_pool.
    name="GEOGRAPHY", index=10, number=22, serialized_options=None, type=None

/Library/Python/3.7/site-packages/google_cloud_bigquery-1.21.0-py3.7.egg/google/cloud/bigquery_v2/proto/standard_sql_pb2.py:83
  /Library/Python/3.7/site-packages/google_cloud_bigquery-1.21.0-py3.7.egg/google/cloud/bigquery_v2/proto/standard_sql_pb2.py:83: DeprecationWarning: Call to deprecated create function EnumValueDescriptor(). Note: Create unlinked descriptors is going to go away. Please use get/find descriptors from generated code or query the descriptor_pool.
    name="NUMERIC", index=11, number=23, serialized_options=None, type=None

/Library/Python/3.7/site-packages/google_cloud_bigquery-1.21.0-py3.7.egg/google/cloud/bigquery_v2/proto/standard_sql_pb2.py:86
  /Library/Python/3.7/site-packages/google_cloud_bigquery-1.21.0-py3.7.egg/google/cloud/bigquery_v2/proto/standard_sql_pb2.py:86: DeprecationWarning: Call to deprecated create function EnumValueDescriptor(). Note: Create unlinked descriptors is going to go away. Please use get/find descriptors from generated code or query the descriptor_pool.
    name="ARRAY", index=12, number=16, serialized_options=None, type=None

/Library/Python/3.7/site-packages/google_cloud_bigquery-1.21.0-py3.7.egg/google/cloud/bigquery_v2/proto/standard_sql_pb2.py:89
  /Library/Python/3.7/site-packages/google_cloud_bigquery-1.21.0-py3.7.egg/google/cloud/bigquery_v2/proto/standard_sql_pb2.py:89: DeprecationWarning: Call to deprecated create function EnumValueDescriptor(). Note: Create unlinked descriptors is going to go away. Please use get/find descriptors from generated code or query the descriptor_pool.
    name="STRUCT", index=13, number=17, serialized_options=None, type=None

/Library/Python/3.7/site-packages/google_cloud_bigquery-1.21.0-py3.7.egg/google/cloud/bigquery_v2/proto/standard_sql_pb2.py:95
  /Library/Python/3.7/site-packages/google_cloud_bigquery-1.21.0-py3.7.egg/google/cloud/bigquery_v2/proto/standard_sql_pb2.py:95: DeprecationWarning: Call to deprecated create function EnumDescriptor(). Note: Create unlinked descriptors is going to go away. Please use get/find descriptors from generated code or query the descriptor_pool.
    serialized_end=590,

/Library/Python/3.7/site-packages/google_cloud_bigquery-1.21.0-py3.7.egg/google/cloud/bigquery_v2/proto/standard_sql_pb2.py:123
  /Library/Python/3.7/site-packages/google_cloud_bigquery-1.21.0-py3.7.egg/google/cloud/bigquery_v2/proto/standard_sql_pb2.py:123: DeprecationWarning: Call to deprecated create function FieldDescriptor(). Note: Create unlinked descriptors is going to go away. Please use get/find descriptors from generated code or query the descriptor_pool.
    file=DESCRIPTOR,

/Library/Python/3.7/site-packages/google_cloud_bigquery-1.21.0-py3.7.egg/google/cloud/bigquery_v2/proto/standard_sql_pb2.py:141
  /Library/Python/3.7/site-packages/google_cloud_bigquery-1.21.0-py3.7.egg/google/cloud/bigquery_v2/proto/standard_sql_pb2.py:141: DeprecationWarning: Call to deprecated create function FieldDescriptor(). Note: Create unlinked descriptors is going to go away. Please use get/find descriptors from generated code or query the descriptor_pool.
    file=DESCRIPTOR,

/Library/Python/3.7/site-packages/google_cloud_bigquery-1.21.0-py3.7.egg/google/cloud/bigquery_v2/proto/standard_sql_pb2.py:159
  /Library/Python/3.7/site-packages/google_cloud_bigquery-1.21.0-py3.7.egg/google/cloud/bigquery_v2/proto/standard_sql_pb2.py:159: DeprecationWarning: Call to deprecated create function FieldDescriptor(). Note: Create unlinked descriptors is going to go away. Please use get/find descriptors from generated code or query the descriptor_pool.
    file=DESCRIPTOR,

/Library/Python/3.7/site-packages/google_cloud_bigquery-1.21.0-py3.7.egg/google/cloud/bigquery_v2/proto/standard_sql_pb2.py:175
  /Library/Python/3.7/site-packages/google_cloud_bigquery-1.21.0-py3.7.egg/google/cloud/bigquery_v2/proto/standard_sql_pb2.py:175: DeprecationWarning: Call to deprecated create function OneofDescriptor(). Note: Create unlinked descriptors is going to go away. Please use get/find descriptors from generated code or query the descriptor_pool.
    fields=[],

/Library/Python/3.7/site-packages/google_cloud_bigquery-1.21.0-py3.7.egg/google/cloud/bigquery_v2/proto/standard_sql_pb2.py:179
  /Library/Python/3.7/site-packages/google_cloud_bigquery-1.21.0-py3.7.egg/google/cloud/bigquery_v2/proto/standard_sql_pb2.py:179: DeprecationWarning: Call to deprecated create function Descriptor(). Note: Create unlinked descriptors is going to go away. Please use get/find descriptors from generated code or query the descriptor_pool.
    serialized_end=602,

/Library/Python/3.7/site-packages/google_cloud_bigquery-1.21.0-py3.7.egg/google/cloud/bigquery_v2/proto/standard_sql_pb2.py:206
  /Library/Python/3.7/site-packages/google_cloud_bigquery-1.21.0-py3.7.egg/google/cloud/bigquery_v2/proto/standard_sql_pb2.py:206: DeprecationWarning: Call to deprecated create function FieldDescriptor(). Note: Create unlinked descriptors is going to go away. Please use get/find descriptors from generated code or query the descriptor_pool.
    file=DESCRIPTOR,

/Library/Python/3.7/site-packages/google_cloud_bigquery-1.21.0-py3.7.egg/google/cloud/bigquery_v2/proto/standard_sql_pb2.py:224
  /Library/Python/3.7/site-packages/google_cloud_bigquery-1.21.0-py3.7.egg/google/cloud/bigquery_v2/proto/standard_sql_pb2.py:224: DeprecationWarning: Call to deprecated create function FieldDescriptor(). Note: Create unlinked descriptors is going to go away. Please use get/find descriptors from generated code or query the descriptor_pool.
    file=DESCRIPTOR,

/Library/Python/3.7/site-packages/google_cloud_bigquery-1.21.0-py3.7.egg/google/cloud/bigquery_v2/proto/standard_sql_pb2.py:236
  /Library/Python/3.7/site-packages/google_cloud_bigquery-1.21.0-py3.7.egg/google/cloud/bigquery_v2/proto/standard_sql_pb2.py:236: DeprecationWarning: Call to deprecated create function Descriptor(). Note: Create unlinked descriptors is going to go away. Please use get/find descriptors from generated code or query the descriptor_pool.
    serialized_end=707,

/Library/Python/3.7/site-packages/google_cloud_bigquery-1.21.0-py3.7.egg/google/cloud/bigquery_v2/proto/standard_sql_pb2.py:263
  /Library/Python/3.7/site-packages/google_cloud_bigquery-1.21.0-py3.7.egg/google/cloud/bigquery_v2/proto/standard_sql_pb2.py:263: DeprecationWarning: Call to deprecated create function FieldDescriptor(). Note: Create unlinked descriptors is going to go away. Please use get/find descriptors from generated code or query the descriptor_pool.
    file=DESCRIPTOR,

/Library/Python/3.7/site-packages/google_cloud_bigquery-1.21.0-py3.7.egg/google/cloud/bigquery_v2/proto/standard_sql_pb2.py:275
  /Library/Python/3.7/site-packages/google_cloud_bigquery-1.21.0-py3.7.egg/google/cloud/bigquery_v2/proto/standard_sql_pb2.py:275: DeprecationWarning: Call to deprecated create function Descriptor(). Note: Create unlinked descriptors is going to go away. Please use get/find descriptors from generated code or query the descriptor_pool.
    serialized_end=792,

/Library/Python/3.7/site-packages/google_cloud_bigquery-1.21.0-py3.7.egg/google/cloud/bigquery_v2/proto/model_pb2.py:54
  /Library/Python/3.7/site-packages/google_cloud_bigquery-1.21.0-py3.7.egg/google/cloud/bigquery_v2/proto/model_pb2.py:54: DeprecationWarning: Call to deprecated create function FileDescriptor(). Note: Create unlinked descriptors is going to go away. Please use get/find descriptors from generated code or query the descriptor_pool.
    google_dot_api_dot_annotations__pb2.DESCRIPTOR,

/Library/Python/3.7/site-packages/google_cloud_bigquery-1.21.0-py3.7.egg/google/cloud/bigquery_v2/proto/model_pb2.py:70
  /Library/Python/3.7/site-packages/google_cloud_bigquery-1.21.0-py3.7.egg/google/cloud/bigquery_v2/proto/model_pb2.py:70: DeprecationWarning: Call to deprecated create function EnumValueDescriptor(). Note: Create unlinked descriptors is going to go away. Please use get/find descriptors from generated code or query the descriptor_pool.
    type=None,

/Library/Python/3.7/site-packages/google_cloud_bigquery-1.21.0-py3.7.egg/google/cloud/bigquery_v2/proto/model_pb2.py:73
  /Library/Python/3.7/site-packages/google_cloud_bigquery-1.21.0-py3.7.egg/google/cloud/bigquery_v2/proto/model_pb2.py:73: DeprecationWarning: Call to deprecated create function EnumValueDescriptor(). Note: Create unlinked descriptors is going to go away. Please use get/find descriptors from generated code or query the descriptor_pool.
    name="RANDOM", index=1, number=1, serialized_options=None, type=None

/Library/Python/3.7/site-packages/google_cloud_bigquery-1.21.0-py3.7.egg/google/cloud/bigquery_v2/proto/model_pb2.py:76
  /Library/Python/3.7/site-packages/google_cloud_bigquery-1.21.0-py3.7.egg/google/cloud/bigquery_v2/proto/model_pb2.py:76: DeprecationWarning: Call to deprecated create function EnumValueDescriptor(). Note: Create unlinked descriptors is going to go away. Please use get/find descriptors from generated code or query the descriptor_pool.
    name="CUSTOM", index=2, number=2, serialized_options=None, type=None

/Library/Python/3.7/site-packages/google_cloud_bigquery-1.21.0-py3.7.egg/google/cloud/bigquery_v2/proto/model_pb2.py:82
  /Library/Python/3.7/site-packages/google_cloud_bigquery-1.21.0-py3.7.egg/google/cloud/bigquery_v2/proto/model_pb2.py:82: DeprecationWarning: Call to deprecated create function EnumDescriptor(). Note: Create unlinked descriptors is going to go away. Please use get/find descriptors from generated code or query the descriptor_pool.
    serialized_end=1230,

/Library/Python/3.7/site-packages/google_cloud_bigquery-1.21.0-py3.7.egg/google/cloud/bigquery_v2/proto/model_pb2.py:97
  /Library/Python/3.7/site-packages/google_cloud_bigquery-1.21.0-py3.7.egg/google/cloud/bigquery_v2/proto/model_pb2.py:97: DeprecationWarning: Call to deprecated create function EnumValueDescriptor(). Note: Create unlinked descriptors is going to go away. Please use get/find descriptors from generated code or query the descriptor_pool.
    type=None,

/Library/Python/3.7/site-packages/google_cloud_bigquery-1.21.0-py3.7.egg/google/cloud/bigquery_v2/proto/model_pb2.py:104
  /Library/Python/3.7/site-packages/google_cloud_bigquery-1.21.0-py3.7.egg/google/cloud/bigquery_v2/proto/model_pb2.py:104: DeprecationWarning: Call to deprecated create function EnumValueDescriptor(). Note: Create unlinked descriptors is going to go away. Please use get/find descriptors from generated code or query the descriptor_pool.
    type=None,

/Library/Python/3.7/site-packages/google_cloud_bigquery-1.21.0-py3.7.egg/google/cloud/bigquery_v2/proto/model_pb2.py:111
  /Library/Python/3.7/site-packages/google_cloud_bigquery-1.21.0-py3.7.egg/google/cloud/bigquery_v2/proto/model_pb2.py:111: DeprecationWarning: Call to deprecated create function EnumValueDescriptor(). Note: Create unlinked descriptors is going to go away. Please use get/find descriptors from generated code or query the descriptor_pool.
    type=None,

/Library/Python/3.7/site-packages/google_cloud_bigquery-1.21.0-py3.7.egg/google/cloud/bigquery_v2/proto/model_pb2.py:114
  /Library/Python/3.7/site-packages/google_cloud_bigquery-1.21.0-py3.7.egg/google/cloud/bigquery_v2/proto/model_pb2.py:114: DeprecationWarning: Call to deprecated create function EnumValueDescriptor(). Note: Create unlinked descriptors is going to go away. Please use get/find descriptors from generated code or query the descriptor_pool.
    name="KMEANS", index=3, number=3, serialized_options=None, type=None

/Library/Python/3.7/site-packages/google_cloud_bigquery-1.21.0-py3.7.egg/google/cloud/bigquery_v2/proto/model_pb2.py:117
  /Library/Python/3.7/site-packages/google_cloud_bigquery-1.21.0-py3.7.egg/google/cloud/bigquery_v2/proto/model_pb2.py:117: DeprecationWarning: Call to deprecated create function EnumValueDescriptor(). Note: Create unlinked descriptors is going to go away. Please use get/find descriptors from generated code or query the descriptor_pool.
    name="TENSORFLOW", index=4, number=6, serialized_options=None, type=None

/Library/Python/3.7/site-packages/google_cloud_bigquery-1.21.0-py3.7.egg/google/cloud/bigquery_v2/proto/model_pb2.py:123
  /Library/Python/3.7/site-packages/google_cloud_bigquery-1.21.0-py3.7.egg/google/cloud/bigquery_v2/proto/model_pb2.py:123: DeprecationWarning: Call to deprecated create function EnumDescriptor(). Note: Create unlinked descriptors is going to go away. Please use get/find descriptors from generated code or query the descriptor_pool.
    serialized_end=6747,

/Library/Python/3.7/site-packages/google_cloud_bigquery-1.21.0-py3.7.egg/google/cloud/bigquery_v2/proto/model_pb2.py:138
  /Library/Python/3.7/site-packages/google_cloud_bigquery-1.21.0-py3.7.egg/google/cloud/bigquery_v2/proto/model_pb2.py:138: DeprecationWarning: Call to deprecated create function EnumValueDescriptor(). Note: Create unlinked descriptors is going to go away. Please use get/find descriptors from generated code or query the descriptor_pool.
    type=None,

/Library/Python/3.7/site-packages/google_cloud_bigquery-1.21.0-py3.7.egg/google/cloud/bigquery_v2/proto/model_pb2.py:145
  /Library/Python/3.7/site-packages/google_cloud_bigquery-1.21.0-py3.7.egg/google/cloud/bigquery_v2/proto/model_pb2.py:145: DeprecationWarning: Call to deprecated create function EnumValueDescriptor(). Note: Create unlinked descriptors is going to go away. Please use get/find descriptors from generated code or query the descriptor_pool.
    type=None,

/Library/Python/3.7/site-packages/google_cloud_bigquery-1.21.0-py3.7.egg/google/cloud/bigquery_v2/proto/model_pb2.py:148
  /Library/Python/3.7/site-packages/google_cloud_bigquery-1.21.0-py3.7.egg/google/cloud/bigquery_v2/proto/model_pb2.py:148: DeprecationWarning: Call to deprecated create function EnumValueDescriptor(). Note: Create unlinked descriptors is going to go away. Please use get/find descriptors from generated code or query the descriptor_pool.
    name="MEAN_LOG_LOSS", index=2, number=2, serialized_options=None, type=None

/Library/Python/3.7/site-packages/google_cloud_bigquery-1.21.0-py3.7.egg/google/cloud/bigquery_v2/proto/model_pb2.py:154
  /Library/Python/3.7/site-packages/google_cloud_bigquery-1.21.0-py3.7.egg/google/cloud/bigquery_v2/proto/model_pb2.py:154: DeprecationWarning: Call to deprecated create function EnumDescriptor(). Note: Create unlinked descriptors is going to go away. Please use get/find descriptors from generated code or query the descriptor_pool.
    serialized_end=6828,

/Library/Python/3.7/site-packages/google_cloud_bigquery-1.21.0-py3.7.egg/google/cloud/bigquery_v2/proto/model_pb2.py:169
  /Library/Python/3.7/site-packages/google_cloud_bigquery-1.21.0-py3.7.egg/google/cloud/bigquery_v2/proto/model_pb2.py:169: DeprecationWarning: Call to deprecated create function EnumValueDescriptor(). Note: Create unlinked descriptors is going to go away. Please use get/find descriptors from generated code or query the descriptor_pool.
    type=None,

/Library/Python/3.7/site-packages/google_cloud_bigquery-1.21.0-py3.7.egg/google/cloud/bigquery_v2/proto/model_pb2.py:172
  /Library/Python/3.7/site-packages/google_cloud_bigquery-1.21.0-py3.7.egg/google/cloud/bigquery_v2/proto/model_pb2.py:172: DeprecationWarning: Call to deprecated create function EnumValueDescriptor(). Note: Create unlinked descriptors is going to go away. Please use get/find descriptors from generated code or query the descriptor_pool.
    name="EUCLIDEAN", index=1, number=1, serialized_options=None, type=None

/Library/Python/3.7/site-packages/google_cloud_bigquery-1.21.0-py3.7.egg/google/cloud/bigquery_v2/proto/model_pb2.py:175
  /Library/Python/3.7/site-packages/google_cloud_bigquery-1.21.0-py3.7.egg/google/cloud/bigquery_v2/proto/model_pb2.py:175: DeprecationWarning: Call to deprecated create function EnumValueDescriptor(). Note: Create unlinked descriptors is going to go away. Please use get/find descriptors from generated code or query the descriptor_pool.
    name="COSINE", index=2, number=2, serialized_options=None, type=None

/Library/Python/3.7/site-packages/google_cloud_bigquery-1.21.0-py3.7.egg/google/cloud/bigquery_v2/proto/model_pb2.py:181
  /Library/Python/3.7/site-packages/google_cloud_bigquery-1.21.0-py3.7.egg/google/cloud/bigquery_v2/proto/model_pb2.py:181: DeprecationWarning: Call to deprecated create function EnumDescriptor(). Note: Create unlinked descriptors is going to go away. Please use get/find descriptors from generated code or query the descriptor_pool.
    serialized_end=6902,

/Library/Python/3.7/site-packages/google_cloud_bigquery-1.21.0-py3.7.egg/google/cloud/bigquery_v2/proto/model_pb2.py:196
  /Library/Python/3.7/site-packages/google_cloud_bigquery-1.21.0-py3.7.egg/google/cloud/bigquery_v2/proto/model_pb2.py:196: DeprecationWarning: Call to deprecated create function EnumValueDescriptor(). Note: Create unlinked descriptors is going to go away. Please use get/find descriptors from generated code or query the descriptor_pool.
    type=None,

/Library/Python/3.7/site-packages/google_cloud_bigquery-1.21.0-py3.7.egg/google/cloud/bigquery_v2/proto/model_pb2.py:199
  /Library/Python/3.7/site-packages/google_cloud_bigquery-1.21.0-py3.7.egg/google/cloud/bigquery_v2/proto/model_pb2.py:199: DeprecationWarning: Call to deprecated create function EnumValueDescriptor(). Note: Create unlinked descriptors is going to go away. Please use get/find descriptors from generated code or query the descriptor_pool.
    name="RANDOM", index=1, number=1, serialized_options=None, type=None

/Library/Python/3.7/site-packages/google_cloud_bigquery-1.21.0-py3.7.egg/google/cloud/bigquery_v2/proto/model_pb2.py:202
  /Library/Python/3.7/site-packages/google_cloud_bigquery-1.21.0-py3.7.egg/google/cloud/bigquery_v2/proto/model_pb2.py:202: DeprecationWarning: Call to deprecated create function EnumValueDescriptor(). Note: Create unlinked descriptors is going to go away. Please use get/find descriptors from generated code or query the descriptor_pool.
    name="CUSTOM", index=2, number=2, serialized_options=None, type=None

/Library/Python/3.7/site-packages/google_cloud_bigquery-1.21.0-py3.7.egg/google/cloud/bigquery_v2/proto/model_pb2.py:205
  /Library/Python/3.7/site-packages/google_cloud_bigquery-1.21.0-py3.7.egg/google/cloud/bigquery_v2/proto/model_pb2.py:205: DeprecationWarning: Call to deprecated create function EnumValueDescriptor(). Note: Create unlinked descriptors is going to go away. Please use get/find descriptors from generated code or query the descriptor_pool.
    name="SEQUENTIAL", index=3, number=3, serialized_options=None, type=None

/Library/Python/3.7/site-packages/google_cloud_bigquery-1.21.0-py3.7.egg/google/cloud/bigquery_v2/proto/model_pb2.py:208
  /Library/Python/3.7/site-packages/google_cloud_bigquery-1.21.0-py3.7.egg/google/cloud/bigquery_v2/proto/model_pb2.py:208: DeprecationWarning: Call to deprecated create function EnumValueDescriptor(). Note: Create unlinked descriptors is going to go away. Please use get/find descriptors from generated code or query the descriptor_pool.
    name="NO_SPLIT", index=4, number=4, serialized_options=None, type=None

/Library/Python/3.7/site-packages/google_cloud_bigquery-1.21.0-py3.7.egg/google/cloud/bigquery_v2/proto/model_pb2.py:211
  /Library/Python/3.7/site-packages/google_cloud_bigquery-1.21.0-py3.7.egg/google/cloud/bigquery_v2/proto/model_pb2.py:211: DeprecationWarning: Call to deprecated create function EnumValueDescriptor(). Note: Create unlinked descriptors is going to go away. Please use get/find descriptors from generated code or query the descriptor_pool.
    name="AUTO_SPLIT", index=5, number=5, serialized_options=None, type=None

/Library/Python/3.7/site-packages/google_cloud_bigquery-1.21.0-py3.7.egg/google/cloud/bigquery_v2/proto/model_pb2.py:217
  /Library/Python/3.7/site-packages/google_cloud_bigquery-1.21.0-py3.7.egg/google/cloud/bigquery_v2/proto/model_pb2.py:217: DeprecationWarning: Call to deprecated create function EnumDescriptor(). Note: Create unlinked descriptors is going to go away. Please use get/find descriptors from generated code or query the descriptor_pool.
    serialized_end=7026,

/Library/Python/3.7/site-packages/google_cloud_bigquery-1.21.0-py3.7.egg/google/cloud/bigquery_v2/proto/model_pb2.py:232
  /Library/Python/3.7/site-packages/google_cloud_bigquery-1.21.0-py3.7.egg/google/cloud/bigquery_v2/proto/model_pb2.py:232: DeprecationWarning: Call to deprecated create function EnumValueDescriptor(). Note: Create unlinked descriptors is going to go away. Please use get/find descriptors from generated code or query the descriptor_pool.
    type=None,

/Library/Python/3.7/site-packages/google_cloud_bigquery-1.21.0-py3.7.egg/google/cloud/bigquery_v2/proto/model_pb2.py:235
  /Library/Python/3.7/site-packages/google_cloud_bigquery-1.21.0-py3.7.egg/google/cloud/bigquery_v2/proto/model_pb2.py:235: DeprecationWarning: Call to deprecated create function EnumValueDescriptor(). Note: Create unlinked descriptors is going to go away. Please use get/find descriptors from generated code or query the descriptor_pool.
    name="LINE_SEARCH", index=1, number=1, serialized_options=None, type=None

/Library/Python/3.7/site-packages/google_cloud_bigquery-1.21.0-py3.7.egg/google/cloud/bigquery_v2/proto/model_pb2.py:238
  /Library/Python/3.7/site-packages/google_cloud_bigquery-1.21.0-py3.7.egg/google/cloud/bigquery_v2/proto/model_pb2.py:238: DeprecationWarning: Call to deprecated create function EnumValueDescriptor(). Note: Create unlinked descriptors is going to go away. Please use get/find descriptors from generated code or query the descriptor_pool.
    name="CONSTANT", index=2, number=2, serialized_options=None, type=None

/Library/Python/3.7/site-packages/google_cloud_bigquery-1.21.0-py3.7.egg/google/cloud/bigquery_v2/proto/model_pb2.py:244
  /Library/Python/3.7/site-packages/google_cloud_bigquery-1.21.0-py3.7.egg/google/cloud/bigquery_v2/proto/model_pb2.py:244: DeprecationWarning: Call to deprecated create function EnumDescriptor(). Note: Create unlinked descriptors is going to go away. Please use get/find descriptors from generated code or query the descriptor_pool.
    serialized_end=7115,

/Library/Python/3.7/site-packages/google_cloud_bigquery-1.21.0-py3.7.egg/google/cloud/bigquery_v2/proto/model_pb2.py:259
  /Library/Python/3.7/site-packages/google_cloud_bigquery-1.21.0-py3.7.egg/google/cloud/bigquery_v2/proto/model_pb2.py:259: DeprecationWarning: Call to deprecated create function EnumValueDescriptor(). Note: Create unlinked descriptors is going to go away. Please use get/find descriptors from generated code or query the descriptor_pool.
    type=None,

/Library/Python/3.7/site-packages/google_cloud_bigquery-1.21.0-py3.7.egg/google/cloud/bigquery_v2/proto/model_pb2.py:266
  /Library/Python/3.7/site-packages/google_cloud_bigquery-1.21.0-py3.7.egg/google/cloud/bigquery_v2/proto/model_pb2.py:266: DeprecationWarning: Call to deprecated create function EnumValueDescriptor(). Note: Create unlinked descriptors is going to go away. Please use get/find descriptors from generated code or query the descriptor_pool.
    type=None,

-- Docs: https://docs.pytest.org/en/latest/warnings.html
============================================== 13 passed, 31 skipped, 100 warnings in 1.13s ==============================================

**Comment by pjsier** [comment]
Sent these notes on setting up a local environment to @shaunagm who mentioned I should add them to this issue.

* +1 to moving `CONTRIBUTING.md` to the top level directory since it was a bit hard to find the contributing guidelines
* The `useful_resources/` directory has some good info for getting folks started, but I didn't see it referenced or linked anywhere else in the documentation
* In the contributing guidelines it says the `sample_code/` folder is at the project root, but it looks like it's now under `useful_resources/`

**Comment by rgriff23** [comment]
@shaunagm here is a list of connectors that definitely need work and which aspects require work. I apologize for the delay, I was worried that my initial audit was a bit too quick and dirty, so I went over them more carefully just now. 

- Action Kit: fix typo in Quick Start (it imports VAN rather than ActionKit)
- Action Network: improve Overview, add Authentication Guide
- Airtable: add Quick Start
- Azure: add Authentication Guide
- Bill.com: improve Overview, add Authentication Guide 
- Braintree: add Authentication Guide
- Civis: add Overview, add Authentication Guide, add Quick Start
- Copper: add Quick Start
- CrowdTangle: improve Overview, add Authentication Guide, add Quick Start
- Databases: Contains multiple classes and needs a global overview at the top to orient the reader. The documentation surrounding individual classes needs to be audited.
- FacebookAds: add Quick Start 
- Freshdesk: improve Overview, add Authentication Guide, add Quick Start
- GitHub: improve Overview
- Google: Contains multiple classes and needs a global overview at the top to orient the reader. The documentation surrounding individual classes needs to be audited.
- Hustle: add Authentication Guide
- Mailchimp: add Authentication Guide
- Mobilize America: add Authentication Guide, add Quick Start
- New/Mode: improve Overview, add Authentication Guide
- PDI: improve Overview (in particular, the entire Overview should not be a single hyperlink)
- Phone2Action: improve Overview, add Authentication Guide, add Quick Start
- Redash: improve Overview, add Authentication Guide, add Quick Start
- Salesfoce: improve Overview, add Authentication Guide, add Quick Start
- SFTP: improve Overview, add Authentication Guide, add Quick Start
- TargetSmart: add Authentication Guide, add Quick Start
- Twilio: add Authentication Guide
- TurboVote: improve Overview, add Authentication Guide, add Quick Start
- Zoom: improve Overview, add Authentication Guide

**Comment by shaunagm** [comment]
Thanks @rgriff23! This is amazingly helpful.

**Comment by davidpablocohn** [comment]
Trying to get started helping, and have been looking at the Google Sheets doc. Setting up/enabling the API on the dev console is non-obvious, but the description of how to do so is probably longer than we want to put in the main doc. Is there an established place for ancillary HowTo docs to go?

**Comment by shaunagm** [comment]
Is there a web page provided by Google that describes the API process in detail? You could do a 1-3 sentence summary and link out to the longer instructions.

**Comment by davidpablocohn** [comment]
Not that I have found (though ironically, Googling Google stuff is often a black hole). The best I've found is a third-party description here: https://www.iperiusbackup.net/en/how-to-enable-google-drive-api-and-get-client-credentials/; I'm writing up a quick summary to see how concise I can get it, and will then shop around for a good location.

**Comment by rgriff23** [comment]
@shaunagm Just making a note here that once #411 is merged (it should be ready), the AWS connectors are done. And with 
@davidpablocohn's PR #417 merged, the Google connectors are done too.

Not sure if you want to close out this issue or keep it open for ongoing improvements to the documentation, but I think the goals outlined in description are achieved! :)

**Comment by shaunagm** [comment]
That's so exciting! Do you have permission to close issues? If so, once 411 is merged, I'll let you do the honors. ;)

**Comment by rgriff23** [comment]
> That's so exciting! Do you have permission to close issues? If so, once 411 is merged, I'll let you do the honors. ;)

Thanks! I am assuming I don't have permissions, because I don't see the 'Close' button (which I think would appear beneath the comment box).
